Upgrade.md

```diff
@@ -133,7 +133,7 @@ class UserListener
 
 ### UserManager
 
-The public methods `refreshUser`, `loadUserByUsername` and `supportsClass` have been
+The public methods `refreshUser`, `loadUserByUsername`, and `supportsClass` have been
```

Comment:
We follow the same rules than the Symfony documentation (https://symfony.com/doc/current/contributing/documentation/standards.html#english-language-standards) so we don't use the serial comma

Reply:
Didn't know that, thank's for info.